usaspending_api/agency/v2/views/bureau_federal_account.py

```diff
@@ -81,24 +81,15 @@ def get_federal_account_list(self) -> List[dict]:
         ]
         last_period_results = (
             GTASSF133Balances.objects.filter(*filters)
-            .filter(*filters)
             .annotate(
                 name=F("treasury_account_identifier__federal_account__account_title"),
                 account_code=F("treasury_account_identifier__federal_account__federal_account_code"),
             )
             .values("name", "account_code")
             .annotate(
-                amount=Sum("total_budgetary_resources_cpe"),
-                unobligated_balance=Sum("budget_authority_unobligated_balance_brought_forward_cpe"),
-                deobligation=Sum("deobligations_or_recoveries_or_refunds_from_prior_year_cpe"),
-                prior_year=Sum("prior_year_paid_obligation_recoveries"),
-            )
-            .annotate(
-                total_budgetary_resources=F("amount") - F("unobligated_balance") - F("deobligation") - F("prior_year"),
-                total_obligations=Sum("obligations_incurred_total_cpe")
-                - Sum("deobligations_or_recoveries_or_refunds_from_prior_year_cpe"),
```

Comment:
are we not subtraction these values anymore because they come from prior years? Some spot checking locally showed higher values in obligations and tbr after the change, which doesn't seem right to me

Reply:
Would expect the values to be higher than they currently are on the page because we are no longer subtracting the prior year values. Reason we would subtract prior year values is in the case of a page similar to COVID Page where we are getting a total across multiple years. (That is my understanding of this issue at least)